CHANGELOG.rst

```diff
@@ -2,6 +2,31 @@
 Changelog
 =========
 
+2.7.11 (2025-01-09)
+===================
+
+* [:backend:`4795`] Fixed not always being able to upload ``.msg`` files.
+* [:backend:`4900`] Fixed submission value variables recoupling for reusable form
+  definitions.
+* [:backend:`4579`] Fixed wrong steps being blocked when logic uses the "trigger from
+  step" option.
+* [:backend:`4825`] Fixed logging to only log empty retrieved data for the
+  authentication flow that is used in the submission.
+* [:backend:`4863`] Fixed authentication flow of an employee via OIDC.
+* Fix tests that I forgot to update 
```

Comment:
```suggestion
```

Reply:
what an idiot, removed it thanks for catching this.